cmd/sample_openapi.yaml

```diff
@@ -0,0 +1,727 @@
+openapi: 3.1.0
+info:
+  title: Petstore - OpenAPI 3.1
+  description: |-
+    This is a sample Pet Store Server based on the OpenAPI 3.1 specification.
+
+    Some useful links:
+    - [OpenAPI Reference](https://www.speakeasyapi.dev/openapi)
+    - [The Pet Store repository](https://github.com/swagger-api/swagger-petstore)
+    - [The source API definition for the Pet Store](https://github.com/swagger-api/swagger-petstore/blob/master/src/main/resources/openapi.yaml)
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: support@speakeasyapi.dev
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
+externalDocs:
+  description: Find out more about Swagger
+  url: http://swagger.io
+security:
+  - api_key: []
+servers:
+  - url: https://{environment}.petstore.io
+    description: A per-environment API.
+    variables:
+      environment:
+        description: The environment name. Defaults to the production environment.
+        default: prod
+        enum:
+          - prod
+          - staging
+          - dev
+tags:
+  - name: pet
+    description: Everything about your Pets
+    externalDocs:
+      description: Find out more
+      url: http://swagger.io
+  - name: store
+    description: Access to Petstore orders
+    externalDocs:
+      description: Find out more about our store
+      url: http://swagger.io
+  - name: user
+    description: Operations about user
+paths:
+  "/pet":
+    put:
+      tags:
+      - pet
+      summary: Update an existing pet
+      description: Update an existing pet by Id
+      operationId: updatePet
+      requestBody:
+        description: Update an existent pet in the store
+        content:
+          application/json:
+            schema:
+              "$ref": "#/components/schemas/Pet"
+        required: true
+      responses:
+        '200':
```

Comment:
Some thoughts:

When I ran `speakeasy quickstart` with python and gave the SDK name `a`:
* `src/a/sdk.py:13:0: C0103: Class name "A" doesn't conform to PascalCase naming style (invalid-name)`
* A bunch of lint warnings were printed ` {"error":"validation warn: [line 384] operation-success-response - operation `createUser` must define at least a single `2xx` `
* These lint warnings were still present when I used '2XX` instead of '200' (as it's now updated to be '200')
* I couldn't get these lint warnings to appear when using `speakeasy openapi validate`, only present if generation failed
* How can we hook into the SDK validation of the SDK name? Or should the python target just be updated to normalize the SDK input to account for enforcing PascalCase?